tests/amazon/test_credentials.py

```diff
@@ -0,0 +1,66 @@
+# coding: spec
+
+from bespin.amazon.credentials import Credentials
+from bespin.errors import BespinError
+from tests.helpers import BespinCase
+from input_algorithms.spec_base import NotSpecified
```

Comment:
I'm a bit nitpicky about imports

Usually I split into groups

    <bespin imports>

    <test specific imports>

    <other imports>

Each block has no empty newlines and is ordered by the length of the line, longest line to shortest line.

so here, would be

    from bespin.amazon.credentials import Credentials
    from bespin.errors import BespinError

    from tests.helpers import BespinCase

    from input_algorithms.spec_base import NotSpecified
    from moto import mock_sts
    import os